demos/index.html

```diff
@@ -14,19 +13,39 @@
         <script type="text/javascript" src="../src/js/dataEntry.js"></script>
         <script type="text/javascript" src="../src/js/dataEntryPanel.js"></script>
         <script type="text/javascript" src="../src/js/chartAuthoring.js"></script>
+        <script type="text/javascript" src="../src/js/d3ViewComponent.js"></script>
+        <script type="text/javascript" src="../src/js/legend.js"></script>
+        <script type="text/javascript" src="../src/js/pie.js"></script>
+        <script type="text/javascript" src="../src/js/pieChart.js"></script>
+        <script type="text/javascript" src="../src/js/d3Utils.js"></script>
+
+        <link rel="stylesheet" type="text/css" href="src/css/chartAuthoring-demo.css" />
 
         <title>Chart Authoring Tool</title>
     </head>
 
     <body>
-        <div id="floec-chartAuthoring"></div>
+        <div id="floec-chartAuthoring">
+            <div id="floec-chartAuthoring-pieChart"></div>
+            <div id="floec-chartAuthoring-dataEntryPanel"></div>
+        </div>
 
         <script type="text/javascript">
             floe.chartAuthoring("#floec-chartAuthoring", {
+                selectors: {
+                    dataEntryPanel: "#floec-chartAuthoring-pieChart",
+                    pieChart: "#floec-chartAuthoring-dataEntryPanel"
+                },
                 templateLoader: {
                     terms: {
                         templatePrefix: "../src/html"
                     }
+                },
+                dataEntryPanel: {
+                    container: "#floec-chartAuthoring-dataEntryPanel"
```

Comment:
The hard-coded id names can be replaced with ```{chartAuthoring}.dom.dataEntryPanel```. This will help to restrict the container to use sub-elements of the parent container.